### DISCUSSION TITLE
How can I associate a Persistent Volume Claim to Airflow for writing to local file system?

### DISCUSSION THREAD

**Discussion by bruciebruce** [apache/airflow#22714] [General]
I have spun up a local RKE2 cluster using Helm charts for most pods and would like to have some Python Airflow Dags persist
data to the local file system.

I have created a persistent volume like this:

apiVersion: v1
kind: PersistentVolume
metadata:
  name: task-pv-volume
  labels:
    type: local
spec:
  storageClassName: manual
  capacity:
    storage: 10Gi
  accessModes:
    - ReadWriteMany
  hostPath:
    path: "/mnt/data"
	
and a persistent volume claim like this:

apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: task-pv-claim
spec:
  storageClassName: manual
  accessModes:
    - ReadWriteMany
  resources:
    requests:
      storage: 3Gi

Question: Where do I couple this claim into the airflow helm chart that I used to deploy airflow?

Thank you

**Comment by ArmyOfNinjas** [comment]
Hi, did you find the answer to this?
